ouroboros-consensus/src/Ouroboros/Consensus/Protocol/ModChainSel.hs

```diff
@@ -67,3 +70,26 @@ instance (Typeable p, Typeable s, ChainSelection p s) => OuroborosTag (ModChainS
 
 instance OuroborosTag p => NoUnexpectedThunks (NodeConfig (ModChainSel p s))
   -- use generic instance
+
+{-------------------------------------------------------------------------------
+  Individual modifiers
+-------------------------------------------------------------------------------}
+
+-- | Override 'preferCandidate' such that the ChainDB will select a new EBB
+-- /promptly/.
```

Comment:
I wouldn't mention ChainDB as this is about chain selection in general.

Reply:
Fixed, thanks.